_partials/developer-materials-results.html.slim

```diff
@@ -41,7 +41,7 @@ div
             | Product
           td.skill Skill Level
       tbody
-        tr.material(ng-repeat="item in data.displayedMaterials" class="get-started-placeholder-{{item.fields.sys_type}}")
+        tr.material(ng-repeat="item in data.displayedMaterials | middlewareOnly" class="get-started-placeholder-{{item.fields.sys_type}}")
```

Comment:
So basically, we just need to add this to the query: `-target_product:(rhel atomic)` to exclude rhel and atomic?